site/pages/docs/index-fr.hbs

```diff
@@ -39,3 +39,8 @@ de l'information comment télécharger, utiliser ou contribuer à la boîte d'ou
 	<li>Suivre le <a href="ref/docstmpl-fr.html">modèle de documentation</a>.</li>
 	<li>Fournir au moins un exemple pour chaque attribut, thème ou élément de la BOEW. D’autres exemples doivent être donnés pour chaque paramètre ou option. </li>
 </ul>
+
+<section>
+	<h2 id="boew31">Avis concernant la BOEW v3.1</h2>
+	<p>À partir du 23 septembre 2014, la version 3.1 de la Boîte à outils de l'expérience Web n'est plus supportée. Le code source et la documentation ont été déplacés vers le dépôt <a href="{{site.legacyrepolink}}">{{site.legacyreponame}}</a>.</p>
+</section>
```

Comment:
Add a blank line to the end of the file (red icon above indicates it is missing).